src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -2025,6 +2148,306 @@ private boolean checkParameters(String wardCode, String chargeCode, String disch
 		return false;
 	}
 	
+	private JComboBox<MedicalType> getJComboMedicalType() {
+		if (medicalTypeComboBox == null) {
+			medicalTypeComboBox = new JComboBox<MedicalType>();
```

Comment:
```suggestion
			medicalTypeComboBox = new JComboBox<>();
```
Don't use the type specification in the constructor call.